src/js/views/views/HallOfFame.js

```diff
@@ -48,15 +48,17 @@ const HallOfFame = ({players}) => {
                 helpers.round(p.careerStats.ewa, 1),
             ],
             classNames: {
-                info: p.statsTids.includes(g.userTid),
+                danger: p.legacyTid === g.userTid,
+                info: p.statsTids.includes(g.userTid) && p.statsTids[p.statsTids.length - 1] !== g.userTid,
```

Comment:
You explicitly ensure info and success are never both set, but danger and info could still be set.